src/components/Summary/AddNewScoreRedux/index.js

```diff
@@ -0,0 +1,133 @@
+import React, { PureComponent } from 'react';
+import { Rate, Row, Col, Divider, Typography } from 'antd';
+
+import reduxForm from 'redux-form/es/reduxForm';
+import Field from 'redux-form/es/Field';
+import { Button, message, Form } from 'antd';
+import createComment from 'utils/comment/comment';
+const { Text, Link } = Typography;
+
+//
+const FieldizeRate = () => ({
+  input,
+  meta,
+  children,
+  hasFeedback,
+  label,
+  ...rest
+}) => {
+  const FormItem = Form.Item;
+  const hasError = meta.touched && meta.invalid;
+  return (
+    <FormItem
+      label={label}
+      validateStatus={hasError ? 'error' : 'success'}
+      hasFeedback={hasFeedback && hasError}
+      help={hasError && meta.error}
+    >
+      <Rate style={{}} allowHalf {...input} {...rest} children={children} />
+    </FormItem>
+  );
+};
+const RateScore = FieldizeRate();
+//Reference: https://codesandbox.io/s/jzyl70wpk?file=/SimpleForm.js:611-1048
+
+class AddNewScoreFormRedux extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = { DisableSubmit: true };
+    this.submitForm = this.props.handleSubmit(async data => {
+      const id = this.props.questionid;
+      const params = {
+        commentRecordId: id,
+        author: data.name,
+        quality: data.quality,
+        hint: data.hint,
+        completeness: data.completeness,
+        tags: 'no more comment',
+        content: data.comment,
+      };
+      await createComment(params);
+      message.success('Add Overall Score successfully');
+      this.props.reset();
+    });
+  }
+  onDetectNullAuthor = e => {
+    if (e.target.value.length === 0) {
+      this.setState({ DisableSubmit: true });
+      console.log(this.state.DisableSubmit);
+    } else {
+      this.setState({ DisableSubmit: false });
+    }
+  };
+  render() {
+    return (
+      <>
+        <Row type="flex" align="middle">
+          <h3 style={{ fontSize: 20, align: 'middle' }}>Interviewer：</h3>
```

Comment:
`fontSize` should have its unit. Such as `20px`. And also, `align` is not a valid CSS property.